{"name": "test_vwologin", "status": "broken", "statusDetails": {"message": "TypeError: selenium.webdriver.remote.webdriver.WebDriver.find_element() argument after * must be an iterable, not NoneType", "trace": "setup = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"0716e44500fd3eebbf8b510f9da1d1db\")>\n\n    @allure.epic(\"VWO Login Test\")\n    @allure.feature(\"TC#1 --> VWO App Positive Test Case\")\n    def test_vwologin(setup):\n        driver = setup\n        loginPage = LoginPage(driver)\n>       loginPage.login_to_vwo(user=\"contact+atb5x@thetestingacademy.com\", pwd=\"ATBx@1234\")\n\ntest_vwologin.py:29: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\PageObjects\\loginPage.py:62: in login_to_vwo\n    self.get_username().send_keys(user)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <tests.PageObjects.loginPage.LoginPage object at 0x000002120A4ABD70>\n\n    def get_username(self):\n>       return self.driver.find_element(*LoginPage.username)\nE       TypeError: selenium.webdriver.remote.webdriver.WebDriver.find_element() argument after * must be an iterable, not NoneType\n\n..\\PageObjects\\loginPage.py:45: TypeError"}, "start": 1711987544653, "stop": 1711987544659, "uuid": "29fa9826-ee42-4ef2-a48d-ad2d0b38032c", "historyId": "c63c120f49a897b020c7de2c2bdebcb5", "testCaseId": "c63c120f49a897b020c7de2c2bdebcb5", "fullName": "tests.vwoLoginTests.test_vwologin#test_vwologin", "labels": [{"name": "feature", "value": "TC#1 --> VWO App Positive Test Case"}, {"name": "epic", "value": "VWO Login Test"}, {"name": "parentSuite", "value": "tests.vwoLoginTests"}, {"name": "suite", "value": "test_vwologin"}, {"name": "host", "value": "DESKTOP-T73FDRD"}, {"name": "thread", "value": "10080-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.vwoLoginTests.test_vwologin"}]}